fuzz/fuzz_targets/fuzz_glob.rs

```diff
@@ -0,0 +1,25 @@
+#![no_main]
+
+extern crate globset;
+extern crate libfuzzer_sys;
+
+use globset::Glob;
+use libfuzzer_sys::fuzz_target;
+use std::str::FromStr;
+
+fuzz_target!(|glob_str: &str| {
```

Comment:
I'd just write this as `libfuzzer_sys::fuzz_target!` and dispense with the import above.